src/Schema.js

```diff
@@ -525,9 +525,7 @@ class Schema {
 
       if (this.data[className][fieldName].type == 'Relation') {
         //For relations, drop the _Join table
-        return database.adaptiveCollection(className).then(collection => {
-          return database.adapter.deleteFields(className, [fieldName], [], database.collectionPrefix, collection);
-        })
+        return database.adapter.deleteFields(className, [fieldName], [])
```

Comment:
can we expose the deleteFields on the dbController just so we don't call adapter here?